### DISCUSSION TITLE
Welcome to wemod-launcher Discussions!

### DISCUSSION THREAD

**Discussion by DaniAsh551** [DeckCheatz/wemod-launcher#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about,  
    if possible open a issue instead of a discussion please.
  * Share ideas.
    This can also be done from the issue tab.  
    For that open a feature request.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
   * We'll be using the discussions in addition to the issue tab

  Hope you have a good experience here.

**Comment by GWTPG** [comment]
is this confirmed to work on Debian 12 stable? when i try to launch it it ends up just crashing a few seconds after pressing play, (when trying to get the installer to launch)

**Reply by marvin1099** [reply]
@DBoys84 this probably works with Debian 12 stable (not tested but verry likely),
most issues come from wayland, 
the other factors usually don't really affect the launcher.
Make shure to follow the guide on the readme,
if you still have issues open a new issue.
